game/scripts/npc/abilities/lycan_summon_wolves.txt

```diff
@@ -39,17 +39,17 @@
       "03"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "wolf_bat"                                        "1.25 1.2 1.15 1.1 1.0 0.9"
+        "wolf_bat"                                        "1.2 1.1 1.0 0.9 0.8 0.7" //OAA, Valve has it wrong
       }
       "04"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "wolf_damage"                                     "18 29 37 46 165 340"
+        "wolf_damage"                                     "26 34 42 50 170 340" //OAA, Valve has it wrong
       }
       "05"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "wolf_hp"                                         "300 350 400 450 1320 2720"
+        "wolf_hp"                                         "300 350 400 450 1500 3000"
```

Comment:
honestly you could probably make it like treants where they start scaling through level 2-4. maybe like 300 400 500 600

Reply:
Will do, after the major.